Source/Atomic/UI/UIEditField.h

```diff
@@ -72,6 +72,10 @@ class UIEditField : public UIWidget
 
     virtual bool OnEvent(const tb::TBWidgetEvent &ev);
 
+    virtual void OnFocusChanged(bool focused);
+
+    // Used to keep track of if we have just been focused for the click select
+    bool firstFocusFlag_;
```

Comment:
Will do, I'll make it private and initialize it.  What's the protocol on bools in C though?  In JS and C# if a bool isn't set it defaults to false.